### ISSUE TITLE
GitLab support?

### ISSUE THREAD

**Issue by NullVoxPopuli** [greenkeeperio/greenkeeper#939]
There are 2 other issues about gitlab support, but I can't seem to find any resolution

**Comment by mblackritter** [comment]
Yeah, I'm looking for the same and it indeed seems weird that there's no movement on GitLab support, especially because users/companies even offered their support on implementation, etc. 😿💦
